app/actions/websocket/index.ts

```diff
@@ -377,6 +379,13 @@ function handleEvent(msg: WebSocketMessage) {
             return dispatch(handleOpenDialogEvent(msg));
         case WebsocketEvents.RECEIVED_GROUP:
             return dispatch(handleGroupUpdatedEvent(msg));
+        case 'custom_com.mattermost.apps_refresh_bindings': {
+            const state = getState();
+            if (appsEnabled(state)) {
+                dispatch(fetchAppBindings(getCurrentUserId(state), getCurrentChannelId(state)));
```

Comment:
Perhaps a handler like the other events so the state is isolated? Easier for the migration later on